jenkins-jobs/cloud-init/integration.yaml

```
@@ -17,19 +17,28 @@
 
 - project:
     name: cloud-init-integration-pytest
+    release:
+      - bionic:
+          release-lts: true
```

bikeshed alert? would is-lts make more sense?
```suggestion
          is-lts: true
```
non-blocking comment

---

I agree; I'll go for `release-is-lts` as I find it more explicit. Thanks for reviewing!